docs/articles/new-expensify/getting-started/Security.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Security
+
+---
```

Comment:
Expensify prioritizes data security and maintains strict compliance standards to safeguard users' sensitive information.